packages/authentication/src/__tests__/unit/fixtures/mock-strategy.ts

```diff
@@ -3,26 +3,35 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Strategy, AuthenticateOptions} from 'passport';
-import {Request} from 'express';
+import {Request} from '@loopback/rest';
+import {AuthenticationStrategy, UserProfile} from '../../../types';
+
+class AuthenticationError extends Error {
```

Comment:
Is `Error` imported from somewhere or a general interface?

Reply:
Ah, you can follow the chat in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/32176, the `Error` interface doesn't have a `code` or `statusCode` property, and using `ErrnoException` violates everywhere that expects an `Error` object, so I created my own Error class to specify the `code`.